docker-compose.yml.dist

```diff
@@ -2,6 +2,7 @@ mail:
   image: tvial/docker-mailserver
   hostname: mail
   domainname: domain.com
+  privileged: true
```

Comment:
Yes, you are right. This is much better. I just read it on:
- https://docs.docker.com/compose/compose-file/#cap-add-cap-drop
- https://groups.google.com/forum/#!topic/docker-user/0e_zsN61Uk0